app/views/user_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= t("app_name") %></h1>
+<p><%= t("hi") %> <%= @user.name %>,</p>
+<p>
+	<%= t "active_msg" %>
+</p>
+<%= link_to t("active"), edit_account_activation_url(id: @user.activation_token, email: @user.email) %>
```

Comment:
Sao lại dùng _url ở đây

Reply:
_path để trả về đường dẫn ngắn ngọn không bao gồm toàn bộ url, _url để trả về đường dẫn tuyệt đối vd: localhost:3000/login